go.mod

```diff
@@ -108,12 +108,12 @@ require (
 	github.com/passbolt/go-passbolt v0.7.2
 	github.com/previder/vault-cli v0.1.2
 	github.com/pulumi/esc-sdk/sdk v0.12.1
-	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.33
+	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.34
 	github.com/sethvargo/go-password v0.3.1
 	github.com/spf13/pflag v1.0.6
 	github.com/tidwall/sjson v1.2.5
-	gitlab.com/gitlab-org/api/client-go v0.133.0
-	k8s.io/kube-openapi v0.0.0-20250701173324-9bd5c66d9911
```

Comment:
It's the kube openapi update. This causes the problem. We will have to tackle this somehow else.

Reply:
Basically, this needs to update together with the other kube apis. Otherwise everything breaks.